test_schools.py

```diff
@@ -159,14 +170,29 @@ def run_test(test: TestInstance, bound: float, plot: bool):
     else:
         if p_val <= 0.05:
             print("\tThe t-stat is not zero or negative, and p-value is of statistical significance (p <= 0.05),"
-                  "the null-hypothesis cannot be rejected.")  # FIXME: does this mean that we can accept the null?
+                  "the null-hypothesis cannot be rejected.")
```

Comment:
When p <= 0.05, we can reject the null-hypothesis, not not reject

Reply:
Fixed in 76f779f